src/Model/MethodGroupGo.cs

```diff
@@ -36,6 +38,17 @@ internal void Transform(CodeModelGo cmg)
         {
             var originalName = Name.Value;
             Name = Name.Value.TrimPackageName(cmg.Namespace);
+
+            // keep a list of all method group names as trimming the package name
+            // can introduce collisions.  if there's a collision append "Group" to
+            // the name.  unfortunately we can't do this in the namer as we don't
+            // have access to the package name.

```

Comment:
ah this sucks. So given this I presume there is no way to put it with the others.

Reply:
That's correct.